apps/s_client.c

```diff
@@ -2859,10 +2858,12 @@ static void print_stuff(BIO *bio, SSL *s, int full)
             /* Redundant if we showed the whole chain */
             if (!(c_showcerts && got_a_chain))
                 PEM_write_bio_X509(bio, peer);
-            X509_NAME_oneline(X509_get_subject_name(peer), buf, sizeof buf);
-            BIO_printf(bio, "subject=%s\n", buf);
-            X509_NAME_oneline(X509_get_issuer_name(peer), buf, sizeof buf);
-            BIO_printf(bio, "issuer=%s\n", buf);
+            BIO_puts(bio, "subject=");
```

Comment:
should we have a utility routine in apps that wraps the prefix/name/suffix?  Something like:
```
app_print(const char* prefix, X509_NAME* n, const char* suffix)
{
print prefix
call name_oneline
print suffix
}
```